### ISSUE TITLE
View freezes until resize

### ISSUE THREAD

**Issue by alvv-z** [diamondburned/dissent#40]
When switching channels (i.e., in the StorageTech server), the loading animation plays, then freezes until the window is resized. I assume this is related to the content of the channel.

Example:
![Example of behavior](https://0x0.st/o1KG.gif)

**Comment by diamondburned** [comment]
The console should have some warnings that can yield why, but it might be what I think it is. Can you paste the console lines that are printed when this happens?

**Comment by alvv-z** [comment]
This is the output:
```
2022/07/12 10:54:51 load channel 749135718701465641
2022/07/12 10:54:51 Discord API: GET /api/v9/channels/749135718701465641/messages
2022/07/12 10:54:51 Warning: Gtk: GtkBox 0x7fae95b1a210 (box) reported min height -12, but sizes must be >= 0
2022/07/12 10:54:51 Warning: Gtk: GtkBox 0x7fae95b1a210 (box) reported min height -12, but sizes must be >= 0
2022/07/12 10:54:51 Warning: Gdk: gdk-frame-clock: layout continuously requested, giving up after 4 tries
2022/07/12 10:54:51 Warning: Gtk: Trying to snapshot GtkApplicationWindow 0x7fae944062b0 without a current allocation
2022/07/12 10:54:51 Warning: Gdk: gdk-frame-clock: layout continuously requested, giving up after 4 tries
2022/07/12 10:54:52 guild 748542142347083868 requested 12 members
```

**Comment by diamondburned** [comment]
That seems to match what I remember. I'll have to spend a day or two squashing out warnings before I can get to this bug, since I've been meaning to do that.

**Comment by diamondburned** [comment]
I think this is fixed as of GTK 4.10. The latest pre-release doesn't seem to exhibit this freezing behavior anymore.

**Comment by diamondburned** [comment]
https://github.com/diamondburned/dissent/commit/928c286e5d57023fceb5ffa3b3cc139f724cca1a should fix this!
